src/vs/workbench/contrib/positronOutputWebview/browser/notebookOutputWebviewServiceImpl.ts

```diff
@@ -304,6 +317,13 @@ export class PositronNotebookOutputWebviewService implements IPositronNotebookOu
 
 		// Create the webview itself
 		const webview = this._webviewService.createWebviewOverlay(webviewInitInfo);
+		webview.onDidDispose(() => { webviewDisposables.dispose(); });
+
+		webviewDisposables.add(webview.onMessage(async ({ message }) => {
```

Comment:
Would this listener better belong in `NotebookOutputWebview` where we have a couple webview listeners/wrappers?

Reply:
This was a bit more work than I expected but I agree and done in https://github.com/posit-dev/positron/pull/4377/commits/29367aeddf1246f5ea2dba61ab7c7a5040a20950